com.VK.Messenger.json

```diff
@@ -0,0 +1,45 @@
+{
+	"id": "com.VK.Messenger",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "24.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "run",
+	"finish-args": [
+		"--socket=x11",
+		"--device=dri",
+		"--share=ipc",
+		"--share=network",
+		"--socket=pulseaudio",
+		"--filesystem=xdg-download"
+	],
+	"modules": [
+		{
+			"name": "vk-messenger",
+			"buildsystem": "simple",
+			"build-commands": [
+				"cp -r vk-messenger /app/bin",
+				"echo -e '#!/bin/sh\n/app/bin/vk-messenger --no-sandbox' > /app/bin/run",
```

Comment:
you should never make a script with `echo`. There is `type: script` source for that.